src/components/stickyFooter/StickyFooter.scss

```diff
@@ -0,0 +1,45 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+$stickyFooterHeight: 58px;
+
+:export { stickyFooterHeight: $stickyFooterHeight; }
+
+.stickyFooter {
+    position: fixed;
+    width: 100%;
+    bottom: 0;
+    right: 0;
+    align-items: center;
+    justify-content: flex-end;
+  
+    display: flex;
+    overflow: hidden;
+  
+    box-shadow: 0 -1px 5px 0 rgba(0,0,0,0.3);
```

Comment:
Footer shouldn't have a shadow, it leads to an incoherent depth of panels. The left nav doesn't have a shadow so does the header and the main panel, so it makes the eye think they're all displayed on the same plane. With the shadow on the footer its like the the admin is under the footer and in the same plane at the same time.

Reply:
@otaillon

Reply:
@gdostie is right, I've been up-UXed. 

Until we revisit the behavior of the header/content/footer family in a more general way, I'd remove the shadow for now.

Reply:
👍

Reply:
removed in the UTs pr https://github.com/coveo/react-vapor/pull/921